locustempus/templates/main/course_detail.html

```diff
@@ -36,22 +36,25 @@
             </div>
         </div>
         {% endif %}
-        <button type="button" class="lt-button mr-1 lt-button--transparent">
-            <span class='lt-icons lt-button__icon'><img src="{{STATIC_URL}}img/icon-grid.svg" class="active" /></span>
-            <span class='lt-button__text'>Grid</span>
-        </button>
-        <button type="button" class="lt-button lt-button--transparent">
-            <span class='lt-icons lt-button__icon'><img src="{{STATIC_URL}}img/icon-list.svg" class="inactive" /></span>
-            <span class='lt-button__text'>List</span>
-        </button>
+        <form method="post">
+            {% csrf_token %}
+            <button type="submit" class="btn lt-button mr-1 lt-button--transparent" {% if project_list_grid %}disabled{% endif %}>
```

Comment:
The grid/list buttons do not disappear when clicked (see below for note on Courses). But...I was a bit confused over what was "active" vs. "inactive". This confusion will likely clear up once the list view is implemented.